Formula/node.rb

```diff
@@ -56,30 +56,11 @@ def install
 
     if build.with? "npm"
       resource("npm").stage buildpath/"npm_install"
-
-      # make sure npm can find node
-      ENV.prepend_path "PATH", bin
```

Comment:
Essentially the way we were installing `npm` from `npm 5.0.0` the only thing it installs is symlinks back to the buildpath. I cannot immediately think of a simple way around this, and believe me, I tried. It's also going to completely mess with the installation of every current `npm`-using formulae.